.env.example

```
@@ -1 +1,3 @@
-REACT_APP_RAPID_API_KEY = <KEY>
+VITE_APP_RAPID_API_KEY=<VALUE>
+VITE_APP_RAPID_API_HOST=<VALUE>
+VITE_APP_RAPID_API_BASE_URL=VALUE>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_APP_RAPID_API_BASE_URL key should go before the VITE_APP_RAPID_API_HOST key

(UnorderedKey)